mobile-sso/js-server-demo/index.js

```diff
@@ -0,0 +1,243 @@
+// server.js
+// where your node app starts
+
+// init project
+if (process.env.NODE_ENV !== 'production') require('dotenv').config()
+
+var express = require('express');
+var app = express();
+var CryptoJS = require('crypto-js');
+var DISQUS_SECRET = process.env.DISQUS_SECRET;
+var DISQUS_PUBLIC = process.env.DISQUS_PUBLIC;
+
+// http://expressjs.com/en/starter/static-files.html
+app.use(express.static('public'));
+
+// http://expressjs.com/en/starter/basic-routing.html
+app.get("/", function (request, response) {
+  response.sendFile(__dirname + '/views/index.html');
+});
+
+// handles login from client and calls SSO code for user in our "database"
+app.get("/login", function (request, response) {
+    console.log("request is", request.query);
+    let username = request.query.username
+    let password = request.query.password
+    let email = request.query.email
+    if (email === 'dmatt+9898@disqus.com') {
+        let payload = disqusSignon(users[0])
+        response.send(payload);
+   } else if (username === 'uniqueperson122327') {
+        let payload = disqusSignon(users[4])
+        response.send(payload);
+   } else {
+        let payload = disqusSignon(users[16])
+        response.send(payload);
+   }
+});
+
+// Simple in-memory "user database" for the purpose of this demo
+var users = [
+  // test user 0 
+  {
+    id:'4226149898',
+    username:'98989898',
+    email:'dmatt+9898@disqus.com',
+    avatar:'https://i.imgur.com/AYgoB04.jpg',
+    url:'https://advrider.com/index.php?members/disqustest.422614/',
+    profile_url:'https://example.com/providedProfileUrl-4226149898'
+  },
+  // test user 1 
+  {
+    id:'123456869',
+    username: 'uniqueperson7',
+    email: 'person7@example.com',
+    avatar:'https://i.imgur.com/AYgoB04.jpg',
+    url: 'example7.com',
+    profile_url:'https://example.com/providedProfileUrl-123456869'
+  },
+  // test user 2 
+  {
+    id:'1234532329',
+    username: 'uniqueperson127',
+    email: 'person327@example.com',
+    avatar:'https://avataaars.io/?avatarStyle=Circle',
+    url: 'example7.com',
+    profile_url:'https://example.com/providedProfileUrl-1234532329'
+  },
+  // test user 3 
+  {
+    id:'1234532329',
+    username: 'uniqueperson126',
+    email: 'person327@example.com',
+    avatar:'https://i.imgur.com/AYgoB04.jpg',
+    url: 'example7.com',
+    profile_url:'https://example.com/providedProfileUrl-1234532329'
+  },
+  // test user 4
+  {
+    id:'1234523232329',
+    username: 'uniqueperson122327',
+    email: 'dmatt@disqus.com',
```

Comment:
Can you remove all the dmatt and taylan example emails and references please.
Does there need to be 16 test users?

Reply:
done